analyzer/consensus/consensus.go

```diff
@@ -175,6 +180,52 @@ func (m *processor) processGenesis(ctx context.Context, chainContext string) err
 	return nil
 }
 
+// Queries oasis-node for its current height.
+func (m *processor) fetchNodeHeights(ctx context.Context) {
+	var (
+		requestCtx       context.Context
+		requestCtxCancel context.CancelFunc = func() {}
+	)
+	for {
+		requestCtxCancel()
+		select {
+		case <-ctx.Done():
+			m.logger.Warn("shutting down node height goroutine", "reason", ctx.Err())
+			return
+		case <-time.After(3 * time.Second):
```

Comment:
we don't use the backoff here because it defaults to running as fast as possible.

Reply:
ok now it's configurable but defaults to 3 seconds

Reply:
Hm I just realized we can generate a response where `latest_indexed_block > latest_block`, because the block analyzer and this fetcher (regardless of whether it lives in its own analyzer) fetch data independently.

Maybe we can work around that by generating the `latest_indexed_block` number as
```sql
SELECT GREATER(
  (SELECT ... FROM chain.latest_node_heights),
  (SELECT ... FROM analysis.processed_blocks WHERE processed_time IS NOT NULL)
)
```
?

Reply:
Good idea; implemented this in the go client instead of the db query though to keep the queries straightforward.